ci/jenkins/jobs/macros.yaml

```diff
@@ -259,3 +259,12 @@
           DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
           chmod +x ./ci/jenkins/test-rancher.sh
           ./ci/jenkins/test-rancher.sh --cluster-name rancher-test --testcase '{conformance_type}' --registry ${{DOCKER_REGISTRY}}
+
+- builder:
+     name: builder-stop-all
+     builders:
+       - shell: |-
+           #!/bin/bash
+           set -e
+           chmod 777 ./ci/jenkins/stop-jobs.sh
```

Comment:
you might want to use "chmod +x" instead of chmod 777 to allow only execute permissions